src/commands/settings/Profile.js

```diff
@@ -37,10 +37,7 @@ module.exports = class Profile extends Command {
                 return ctx.editMessage({ content: null, embeds: [embed] });
             }
             else {
-                let bio;
-                if (!u.bio) bio = 'No bio set. To set your bio type `ear bio <desired bio>`';
-                else bio = u.bio;
-
+                const bio = u ? u.bio : 'No bio set. To set your bio type `ear bio <desired bio>`';
```

Comment:
The if statement above already checks if there is a user. This should be changed to `u.bio ? u.bio : '...';`

Reply:
True, didn't think of that. I will change that to nullish coalescing operator